### PR TITLE
Refactored useSpringScroll to remove unnecessary uses of useMemo

### PR DESCRIPTION
**Pull request by rijk**
This is not super important, but while trying to fix something else (I'll open up an issue for it later) I came across this piece of code which I thought would be a little more legible like this.

### CODE REVIEW

**Comment by breadadams** [comment]
Thanks for this @rijk, and sorry for the delay in getting back to you on the issues I saw you opened - been swamped with work...

I'll be looking at it all next week 👍

File: src/hooks/useScrollerMotion/useSpringScroll.ts [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
 import { MotionValue, SpringProp, SpringConfig } from '@/types'
 
 const useSpringScroll = (springConfig: SpringProp): MotionValue => {
   const { scrollY } = useViewportScroll()
 
-  const enableSpring = useMemo(() => typeof springConfig === 'object', [
-    springConfig
-  ])
+  const config = typeof springConfig === 'object' ? springConfig : undefined
+  const spring = useSpring(scrollY, config as SpringConfig | undefined)
```
Comment (gabrielperales) [inline src/hooks/useScrollerMotion/useSpringScroll.ts:9]: the last `undefined` won't be necessary

**Comment by gabrielperales** [comment]
@breadadams looks good to me... there aren't complex calculations to need `useMemo`

**Review by breadadams — APPROVED** [review]

**Comment by breadadams** [comment]
Btw @rijk if you want to provide an email/website we can get you in the package.json contributors list 🙂, or feel free to open a PR with it yourself.

**Comment by rijk** [comment]
Oh, it's not necessary for me, but  you can use rijk@uniti.be if you want

**Pull request merged**